PCbuild/_freeze_module.vcxproj

```diff
@@ -91,11 +91,13 @@
       <PreprocessorDefinitions>Py_NO_ENABLE_SHARED;Py_BUILD_CORE;_CONSOLE;%(PreprocessorDefinitions)</PreprocessorDefinitions>
       <Optimization>Disabled</Optimization>
       <WholeProgramOptimization>false</WholeProgramOptimization>
+      <InlineFunctionExpansion>Disabled</InlineFunctionExpansion>
```

Comment:
This should be the default, but why does it matter for `_freeze_module.vcxproj` at all? This app is only used during build.

Reply:
Few people debugs `_freeze_module`, so the low stack usage by un-inlining would be safer. 
If we pass other inlining options like /Ob3 manually to `_freeze_module` by mistake, it overrides the default option and changes the stack usage. `Disabled` option emits D9025 warning annoyingly, `Default` not.

Reply:
`_freeze_module.vcxproj` are reverted since it does not hit the threshold (2,000,000 bytes) on its job at this point even if it is an unoptimized 64bit executable.